jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/resources/META-INF/CaseMgmtorm.xml

```diff
@@ -478,7 +478,7 @@
                 t.taskData.processInstanceId,
                 t.taskData.parentId,
                 t.taskData.deploymentId,
-                t.taskData.skipable              )
+                t.taskData.skipable, plog.correlationKey, plog.processType)
```

Comment:
Same here: It should be "log" instead of "plog" as alias
```suggestion
                t.taskData.skipable, log.correlationKey, log.processType)
```